CustomerData/PrivateData.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Metrilo\Analytics\CustomerData;
+
+use Magento\Customer\CustomerData\SectionSourceInterface;
+
+class PrivateData implements SectionSourceInterface
+{
+    public function __construct(
+        \Metrilo\Analytics\Helper\SessionEvents $sessionEvents
+    ) {
+        $this->sessionEvents = $sessionEvents;
+    }
+    
+    public function getSectionData()
+    {
+        $events = $this->sessionEvents->getSessionEvents();
```

Comment:
does this remove the events from the session?

Reply:
Yes.